spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/TestSparkDistributionAndOrderingUtil.java

```diff
@@ -43,7 +44,9 @@
 import org.apache.spark.sql.connector.write.RowLevelOperation.Command;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
 
+@ExtendWith(ParameterizedTestExtension.class)
```

Comment:
There is no parameterized test previously?

Reply:
This is the parameterized test previously as well. While the super class already has the `@ExtendWith`, this line should be added to clearly incicate this is a parameterized test class.